docs/ru/raspberry.md

```diff
@@ -1,9 +1,9 @@
 Raspberry Pi
 ============
 
-**Raspberry Pi** – это свободно помещающийся на ладони одноплатный компьютер, созданный на базе мобильного микропроцессора ARM11. У него низкое потребление энергии, и он может работать даже от солнечных батарей. Raspberry Pi 3 входит в комплект программируемого квадрокоптера "Клевер 2", "Клевер 3" и "Клевер 4".
+**Raspberry Pi** – это свободно помещающийся на ладони одноплатный компьютер, созданный на базе мобильного микропроцессора ARM11. У него низкое потребление энергии, и он может работать даже от солнечных батарей. Raspberry Pi входит в комплекты программируемого квадрокоптера Клевер.
```

Comment:
It's not using ARM11 since its second iteration (the Raspberry Pi 2 series, except for the Zero series). ARM11 is not a mobile microprocessor, it's a microarchitecture. Just mention `arm` as an instruction set.

Reply:
Please, fix.